LibVLCSharp/LibVLCSharp.csproj

```diff
@@ -56,6 +56,9 @@ LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.</Descri
   <PropertyGroup Condition="$(NETSTANDARD_ONLY)=='true'">
     <TargetFrameworks>netstandard2.0;netstandard1.1</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
```

Comment:
I guess this is fine, but since it's not (supposed to be) doing anything with any TFM other than the UAP one, you could have use the TFM in the condition instead of `IsOsPlatform`. As you wish though, no strong feelings here.

Reply:
You are right, I changed it (cf d9d745ab2d185dfad7279e1f19a935b4750a34d6).